dissect/hypervisor/backup/vma.py

```diff
@@ -206,20 +215,19 @@ def _read(self, offset, length):
         return b"".join(result)
 
 
-def _iter_clusters(vma, dev_id, cluster, count):
+def _iter_clusters(vma, device, cluster, count):
```

Comment:
The change from `dev_id` to `device` is not really necessary, it makes it a little less clear IMO.